src/main/java/com/puppycrawl/tools/checkstyle/JavadocDetailNodeParser.java

```diff
@@ -50,6 +51,29 @@
 public class JavadocDetailNodeParser {
 
     /**
+     * Options for errors listeners attached to ANTLR parser.
+     */
+    public enum ErrorListenerOption {
+
+        /**
+         * Default option for error listeners attached to ANTLR parser. Error listeners will not
+         * be notified of non-tight HTML in javadoc when this option is supplied to
+         * {@link #parseJavadocAsDetailNode(DetailAST, ErrorListenerOption)} as its
+         * second argument.
+         */
+        DEFAULT,
```

Comment:
What does it mean "default" ?

It is a prarser option , not a listners.

Should be:
ParserErrorOption {VIOLATE_NON_TIGHT_HTML, ACCEPT_NO_NON_TIGHT_HTML}